int32 value
---
bool return